src/main/webapp/app/admin/logs/logs.component.ts

```diff
@@ -8,22 +8,19 @@ import { LogsService } from 'app/admin/logs/logs.service';
     styleUrls: ['./logs.component.scss'],
 })
 export class LogsComponent implements OnInit {
-    loggers: Log[];
-    filter: string;
-    orderProp: string;
-    reverse: boolean;
+    loggers?: Log[];
+    filteredAndOrderedLoggers?: Log[];
+    filter = '';
+    orderProp: keyof Log = 'name';
+    ascending = true;
 
```

Comment:
Does it make sense to refactor this more so these attributes so that one doesn't have to define them every time or would you consider this overkill?

Reply:
Not necessary since generated from jhipster